processing/src/main/java/io/druid/segment/DoubleColumnSelector.java

```diff
@@ -26,5 +26,32 @@
 public interface DoubleColumnSelector extends ColumnValueSelector, HotLoopCallee
 {
   @CalledFromHotLoop
-  double get();
+  @Override
+  double getDouble();
```

Comment:
`asDouble()` is "wrong" because by convention it means that that selector object is converted to double. So probably methods should be called `getAsDouble()`, `getAsLong()` and `getAsFloat()`. But it creates inconsistency with `Aggregator` interface. Also "As" method names are longer, and I hardly think anyone could be confused by the current names (e. g. think about them as they have `ByteBuffer.getDouble()` semantics). So I'm slightly pro keeping the current names.

Regarding `get()` overload, I specifically wanted to remove conflict between `Double/Float/LongColumnSelector` interfaces, so that some class could implement all three. It could simplify implementation of some ColumnValueSelector factories. Also, I don't see why having two methods with different names but the same semantics is better than having just one.